### ISSUE TITLE
Getting the same evaluation metrics with completely different covariates

### ISSUE THREAD

**Issue by sparkey63** [unit8co/darts#531]
Hi,

I've been using darts for just over a week now and it's great, but I've been trying to work out which of my covariates in the dataset help to improve the accuracy of the prediction but I keep getting exactly the same R2, MAPE and SMAPE scores on completely separate covariates, is there something I'm missing on how the model saves and do I need to rename the model with model_name to avoid it using some saved configuration?

Thanks

**Comment by hrzn** [comment]
Hi @sparkey63, do you still have this issue? If yes, a few lines of code would be helpful to try and reproduce.

**Comment by maheshs11** [comment]
> Hi @sparkey63, do you still have this issue? If yes, a few lines of code would be helpful to try and reproduce.

model = NBEATSModel(
    input_chunk_length=10,
    output_chunk_length=1,

)
model.load_model('new.pth.tar')
model.fit(train)

every time we load a model should we call fit and give parameter?
could u give more info regarding this?

**Comment by hrzn** [comment]
> > Hi @sparkey63, do you still have this issue? If yes, a few lines of code would be helpful to try and reproduce.
> 
> model = NBEATSModel( input_chunk_length=10, output_chunk_length=1,
> 
> ) model.load_model('new.pth.tar') model.fit(train)
> 
> every time we load a model should we call fit and give parameter? could u give more info regarding this?

I don't think this is linked to this thread and that this code reproduces the issue of @sparkey63. Please open a separate issue if you need to address a specific topic.
